docs/en/observability/quickstarts/monitor-hosts-with-otel.asciidoc

```diff
@@ -0,0 +1,63 @@
+[[quickstart-monitor-hosts-with-otel]]
+= Quickstart: Monitor hosts with OpenTelemetry
+
+preview::[]
+
+In this quickstart guide, you'll learn how to collect logs and metrics using the Elastic Distribution of OpenTelemetry (EDOT) Collector.
```

Comment:
This sentence is very different from what the title says. Collecting logs and metrics is not always the same as monitoring hosts.